frontend/src/components/SubmitArtComponent.vue

```diff
@@ -48,54 +56,96 @@ import Checkbox from 'primevue/checkbox'
 import ScrollPanel from 'primevue/scrollpanel'
 import Button from 'primevue/button'
 import { useImageStore } from '../stores/images'
-import { useUserStore } from '@/stores/user'
+import { useUserStore } from '../stores/user'
+import Toast from 'primevue/toast'
+import { useToast } from 'primevue/usetoast'
 
 const imageStore = useImageStore()
 const checked = ref(false)
 const type = ref('unscreened')
-const link = ref('')
-const message = ref('')
-const file = ref<File>()
+const links = ref([''])
+const files = ref<File[]>([])
 const uploading = ref(false)
-const ok = computed(() => checked.value && file.value && !uploading.value)
+const ok = computed(() => checked.value && files.value && !uploading.value)
+const pictures = ref(1)
+const toast = useToast()
 
 const userStore = useUserStore()
 const user = userStore.currentUser
+const isAdmin = userStore.isAdmin
 
 async function uploadedFile(e: FileUploadSelectEvent) {
-  file.value = e.files[0]
-  message.value = 'File successfully uploaded'
+  files.value.push(e.files[0])
+  console.log(files.value)
+  toast.add({
+    severity: 'success',
+    summary: 'Success',
+    detail: 'File successfully uploaded.',
+    life: 3000
+  })
 }
 
 async function submit() {
   try {
     uploading.value = true
     if (!user) {
-      message.value = `You must be logged in to submit art`
+      toast.add({
+        severity: 'warn',
+        summary: 'Warning',
+        detail: 'You must be logged in to submit art.',
+        life: 3000
+      })
       throw new Error('not logged in')
     }
 
-    if (!file.value) {
-      message.value = "You didn't attach a file"
+    if (files.value.length != links.value.length) {
+      console.log(files.value.length, links.value.length)
+      toast.add({
+        severity: 'warn',
+        summary: 'Warning',
+        detail: "You didn't attach a file.",
+        life: 3000
+      })
       throw new Error('there is no file in ba sing se')
     }
+    console.log(links.value.length)
 
-    const formData = new FormData()
-    formData.append('type', type.value)
-    formData.append('link', link.value)
-    formData.append('image', file.value)
-
-    const res = await imageStore.uploadImage(formData)
-    if (res?.ok) location.reload()
-    else throw new Error((await res.json()).error)
+    for (let i = 0; i < links.value.length; i++) {
+      const formData = new FormData()
+      formData.append('type', type.value)
+      formData.append('link', links.value[i])
+      formData.append('image', files.value[i])
+      console.log(formData)
+      const res = await imageStore.uploadImage(formData)
```

Comment:
this is uploading a single image multiple times; i think you can send multiple images to the endpoint all at once now (don't quote me on that) (i was using postman)